extensions/flyway/deployment/src/test/resources/migrate-at-start-config.properties

```diff
@@ -1,6 +1,10 @@
-quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:test_quarkus;DB_CLOSE_DELAY=-1
+quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:test_quarkus;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'src/test/resources/h2-init-data.sql'
 quarkus.datasource.driver=org.h2.Driver
 quarkus.datasource.username=sa
 quarkus.datasource.password=sa
 # Flyway config properties
-quarkus.flyway.migrate-at-start=true
\ No newline at end of file
+quarkus.flyway.migrate-at-start=true
+quarkus.flyway.table=test_flyway_history
+quarkus.flyway.baseline-on-migrate=true
+quarkus.flyway.baseline-version=0.0.1
+quarkus.flyway.baseline-description=Initial description for test
```

Comment:
Any chance you could create a separate test instead of changing the existing one?

I think it would be better.

Reply:
Ok, I will split them